lib/vagrant-libvirt/action/create_domain_volume.rb

```diff
@@ -53,6 +53,7 @@ def call(env)
                     xml.permissions do
                       xml.owner storage_uid(env)
                       xml.group storage_gid(env)
+                      xml.mode '0744'
```

Comment:
This suggests that maybe virt-v2v-copy-to-local should be used first to retrieve the files from libvirt, as making things readable by other users will likely be considered a security risk.